packages/twenty-front/src/modules/activities/emails/components/EmailThreadMessage.tsx

```diff
@@ -22,21 +22,52 @@ const StyledThreadMessageHeader = styled.div`
 `;
 
 type EmailThreadMessageProps = {
-  id: string;
   body: string;
   sentAt: string;
-  from: MockedEmailUser;
-  to: MockedEmailUser[];
+  participants: EmailThreadMessageParticipant[];
+};
+
+const getDisplayNameFromParticipant = (
+  participant: EmailThreadMessageParticipant,
+) => {
+  if (participant.displayName) {
```

Comment:
I think we want to retrieve the name from person/workspaceMember first before fallback to the email displayName